x-pack/test/apm_api_integration/tests/mobile/mobile_http_requests_timeseries.spec.ts

```diff
@@ -125,8 +125,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expect(response.status).to.be(200);
         expect(ntcCell.status).to.be(200);
-        expect(response.body.currentPeriod.timeseries[0].y).to.eql(0);
-        expect(ntcCell.body.currentPeriod.timeseries[0].y).to.eql(0);
+        expect(response.body.currentPeriod.timeseries[0].y).to.eql(2);
+        expect(ntcCell.body.currentPeriod.timeseries[0].y).to.eql(1);
```

Comment:
When generating data we create to instances for `wifi` and 1 for `cell`, not sure why the test were passing before, now with the improvements in the query seems to return proper data